homework/index.js

```diff
@@ -30,13 +30,95 @@
     return elem;
   }
 
+  function renderContributors(contributors, contributorsContainer) {
+    createAndAppend('p', contributorsContainer, { text: 'Contributors', id: 'contributors' });
+    contributors.forEach(contributor => {
+      const contributorsDetailsDiv = createAndAppend('div', contributorsContainer, {
+        class: 'contributor-details',
+      });
+      createAndAppend('img', contributorsDetailsDiv, {
+        src: contributor.avatar_url,
+        alt: contributor.login,
+        class: 'image',
+      });
+      createAndAppend('a', contributorsDetailsDiv, {
+        text: contributor.login,
+        href: contributor.html_url,
+        target: '_blank',
+        class: 'contributor-data',
+      });
+      createAndAppend('p', contributorsDetailsDiv, {
+        text: contributor.contributions,
+        class: 'contributor-badge',
+      });
+    });
+  }
+
+  function fetchAndRenderData(selectedRepo, repoContainer, contributorsContainer, root) {
+    repoContainer.innerHTML = '';
+    contributorsContainer.innerHTML = '';
+
+    const updatedAt = new Date(selectedRepo.updated_at);
+    createAndAppend('p', repoContainer, {
```

Comment:
The assignment asks that the repo name is a link that opens the current GitHub repo in a new browser tab.